test/cql-pytest/test_permissions.py

```diff
@@ -348,6 +348,8 @@ def test_grant_revoke_uda_permissions(scylla_only, cql):
                         def create_aggr_idempotent():
                             user_session.execute(f"CREATE AGGREGATE IF NOT EXISTS {keyspace}.{custom_avg} {custom_avg_body}")
                             cql.execute(f"DROP AGGREGATE IF EXISTS {keyspace}.{custom_avg}(bigint)")
+                        grant(cql, 'EXECUTE', f'function {keyspace}.{avg_partial}(tuple<bigint, bigint>, bigint)', username)
```

Comment:
Here too, I don't understand how this test passed on Cassandra before this patch. I supposed Cassandra had this bug as well? Maybe this should have been mentioned in the commit message.

Reply:
This test "passed" before because it was tagged `scylla_only`, without it it would not pass.
I'm not sure why the test was written only for Scylla, considering that most tests next to it have an implementation for both Scylla and Cassandra - maybe it's because it has a more complicated Lua function (working on tuples) to translate to Java. I can add the missing implementation together with the rest of Cassandra-targeted fixes

Reply:
I guess that if it was already marked "scylla_only" then it's a pre-existing problem and you don't have to fix it now (on the other hand, if you don't fix it now, it's unlikely anyone will ever remember to fix it ;-)).
So would you like me to merge this PR as-is, or you want to send a new version without the "scylla_only"?

Reply:
By the way, you mentioned in a response to Avi above that in a specific case "The user can still execute the UDA. It's arguable whether this is the best approach, but the same thing happens in Cassandra.". A test could be added for this too, and then it will be easy to show that it behaves like Cassandra.

Reply:
I think I would prefer to send the "scylla_only" fixes in another patch, so we can try merging this one

Reply:
Merged